common/app/conf/switches/FeatureSwitches.scala

```diff
@@ -464,4 +464,13 @@ trait FeatureSwitches {
     exposeClientSide = false,
   )
 
+  val NewslettersRemoveConfirmationStep = Switch(
+    SwitchGroup.Feature,
+    "newsletters-remove-confirmation-step",
+    "Remove confirmation step when user sign up to a newsletter",
+    owners = Seq(Owner.withEmail("newsletters.dev@guardian.co.uk")),
+    safeState = Off,
+    sellByDate = LocalDate.of(2022, 1, 14),
```

Comment:
I think this is the date the switch expires (and becomes always "off"). For this feature, I think we should use a `sellByDate` of `None`, meaning this switch should not automatically expire.

Reply:
set to never